src/framework/piral-base/src/load.test.ts

```diff
@@ -36,7 +36,6 @@ describe('Loading Modules', () => {
     const promise = loadMetadata(() =>
       Promise.resolve([data]),
     );
-    expect(promise).rejects.toBeNull();
```

Comment:
Any reason for removing those?

Reply:
I don't think `...rejects.toBeNull()` is a common pattern.  Sort of asserting that "there should be no rejections" (?).  That assertion was throwing an error during the test -- something about the promise actually being set up to be resolved instead of rejected.

Further down, there are other assertions that check against the resolved value, which would fail anyway if the promise was really rejected.